easybuild/easyconfigs/t/TopHat/TopHat-2.1.1-goolf-1.7.20.eb

```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'ConfigureMake'
+
+name = 'TopHat'
+version = '2.1.1'
+
+homepage = 'http://ccb.jhu.edu/software/tophat/'
+description = """TopHat is a fast splice junction mapper for RNA-Seq reads."""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+toolchainopts = {'pic': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://ccb.jhu.edu/software/tophat/downloads/']
+
+patches = [
+    'tophat-2.0.13-zlib.patch',
+]
+
+dependencies = [
+    ('Boost', '1.58.0', '-Python-2.7.10'),
```

Comment:
@rjeschmi do you want to stick to Boost 1.58.0 here, or shall we bump this to 1.60.0 too to have both TopHat easyconfigs in sync?